modules/local/dada2_addspecies.nf

```diff
@@ -36,9 +36,22 @@ process DADA2_ADDSPECIES {
     set.seed(100) # Initialize random number generator for reproducibility
 
     taxtable <- readRDS(\"$taxtable\")
-    taxa <- addSpecies(taxtable, \"$database\", $options.args, verbose=TRUE)
-    # Put the sequence last
-    taxa <- taxa[,c(colnames(taxa)[!colnames(taxa) %in% 'sequence'], 'sequence')]
+    t <- addSpecies(taxtable, \"$database\", $options.args, verbose=TRUE)
```

Comment:
I somewhat dislike the variable name `t` because `t(x)` transposes the data table x. But thats maybe just me.

Reply:
You have a point. I use it regularly for temporary t(ables). I'll update.